python/cugraph-pyg/cugraph_pyg/nn/conv/base.py

```diff
@@ -18,26 +18,12 @@
 
 torch = import_optional("torch")
 torch_geometric = import_optional("torch_geometric")
-
-try:  # pragma: no cover
-    from pylibcugraphops.pytorch import CSC, HeteroCSC
-
-    HAS_PYLIBCUGRAPHOPS = True
-except ImportError:
-    HAS_PYLIBCUGRAPHOPS = False
+ops_torch = import_optional("pylibcugraphops.pytorch")
 
 
 class BaseConv(torch.nn.Module):  # pragma: no cover
     r"""An abstract base class for implementing cugraph-ops message passing layers."""
 
-    def __init__(self):
-        super().__init__()
```

Comment:
Is this deletion intention ?

Reply:
Yes, linter thinks this is redundant, as it does not add anything on top of `__init__()` of base class.

Reply:
Gotcha,  I am unsure if we should remove the `__init__` call for derived classes. Maybe cause I have not seen it. 

CC: @rlratzel/ @eriknw  , in case we have opinions on calling the constructor of the base class implicitly/explicitly.

Reply:
Defining `__init__` here does seem unnecessary, since it doesn't change any arguments to `__init__` nor does it do anything else at all any more since it no longer checks whether `pylibcugraphops.pytorch` exists.

Reply:
Thanks for the guidance. 

 I think this PR is ready to merge now.